### ISSUE TITLE
Document usage

### ISSUE THREAD

**Issue by xurizaemon** [jitendrapurohit/nz.co.fuzion.deleteoldbulkmailings#4]
I haven't tested the "usage" instructions, and in CiviCRM chat I thought I saw Pete say you have to turn off logging (otherwise this extension will create new records as it tries to clean up?).

But in the code I see you [disable logging before doing the cleanup](https://github.com/jitendrapurohit/nz.co.fuzion.deleteoldbulkmailings/blob/a276f959f69c66441e22b442cd50409d2d81833b/api/v3/Bulkmailing/Deleteoldrecords.php#L12), so ...

Is it necessary to turn off CiviCRM's change logging before using this? If so, that should be documented.

**Comment by jitendrapurohit** [comment]
>Is it necessary to turn off CiviCRM's change logging before using this? If so, that should be documented.

Nope - that is not necessary as you've seen we've handled this in the code. But maybe, we should handle it more appropriately. It seems it enables the logging even if the site was having it disabled before the job execution. I'll test that soon and update.

Thanks

**Comment by adixon** [comment]
Yes, it'll turn on logging if you start with it off. Which is a pretty big deal on a large install.

But otherwise - nice extension, just what I was going to write myself until I found yours, thanks!
